Core/Core/Features/Planner/CalendarMain/CalendarDaysViewController.swift

```diff
@@ -115,11 +115,12 @@ class CalendarDaysViewController: UIViewController {
     }
 
     override func traitCollectionDidChange(_ previousTraitCollection: UITraitCollection?) {
-        super.traitCollectionDidChange(previousTraitCollection)
-
-        // Manually refresh the spacing upon rotation
-        if traitCollection.verticalSizeClass != previousTraitCollection?.verticalSizeClass {
-            weeksStackView.spacing = weekGap
+        let traits = [UITraitVerticalSizeClass.self]
+        registerForTraitChanges(traits) { (self: Self, previousTraitCollection: UITraitCollection) in
+            // Manually refresh the spacing upon rotation
+            if self.traitCollection.verticalSizeClass != previousTraitCollection.verticalSizeClass {
+                self.weeksStackView.spacing = self.weekGap
+            }
```

Comment:
This subscription should only happen once and not every invocation of the `traitCollectionDidChange` method.